### ISSUE TITLE
[Feature Request] select non-consective/concurrent lines for highlight/copying

### ISSUE THREAD

**Issue by georgesnow** [subethaedit/SubEthaEdit#146]
it would be nice to have the ability to highight/select non-consecutive lines and be able to cut/copy those lines. 

Examples:

1. a
2.b
3.c
4.d
5.e

I could highlight/select lines 2,4, and 5 and copy/cut them.

**Comment by monkeydom** [comment]
Indeed. Currently multiple selection is not a thing, only blockedit. This is related to #68 

Things that work, but are weird and not a replacement for that feature reqeust: The find all window can multiple select and copy the found lines. So you can do a search for `[^\n\r]+` and command select in the find all window to extract just certain lines.
